### PR TITLE
add /commands command

### PR DESCRIPTION
**Pull request by vdbg**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -225,10 +228,14 @@ def request_access(self, update: Update, context: CallbackContext, access_level:
             self.command_unknown(update, context)
             raise PermissionError(f"UserId {update.effective_user.id}, handle {update.effective_user.name} is attempting level {access_level} command without permission.")
 
-    def device_actuator(self, update: Update, context: CallbackContext, command, message: str) -> None:
+    def device_actuator(self, update: Update, context: CallbackContext, command: Union[str, list], bot_command: str, message: str) -> None:
         self.request_access(update, context, AccessLevel.DEVICE)
         device = self.get_device(update, context)
         if device:
+            supported_commands = device['supported_commands']
+            if not bot_command in supported_commands:
```
Comment (Nickolaim) [inline main.py:236]: Another way to write it that's more English is `if bot_command not in supported_commands:`

File: main.py [hunk 2]
```diff
@@ -311,33 +321,43 @@ def command_unknown_user(self, update: Update, context: CallbackContext) -> None
...
         text.append("```")
         self.send_md(update, context, text)
 
+    def get_percent(self, input: str) -> int:
+        percent = -1
+        try:
+            percent = int(input)
+        except ValueError:
+            return None
+        if percent < 0 or percent > 100:
```
Comment (Nickolaim) [inline main.py:346]: 3 lines can be replaced with `return percent if 100 > percent > 0 else None`

File: main.py [hunk 3]
```diff
@@ -311,33 +321,43 @@ def command_unknown_user(self, update: Update, context: CallbackContext) -> None
...
+            return None
+        return percent
+
     def command_dim(self,  update: Update, context: CallbackContext) -> None:
         self.request_access(update, context, AccessLevel.DEVICE)
         if len(context.args) < 2:
-            self.send_text("Dim level and device name must be specified.")
+            self.send_text(update, context, "Dim level and device name must be specified.")
```
Comment (Nickolaim) [inline main.py:353]: is this assumption correct that any 2 parameters specified are have to be level and device name?  What if there are 3 params?

Comment (vdbg) [reply main.py:353]: the bot library separates cmd args with spaces.
therefore,
/dim 20 downstairs office light
will show as 4 args.
so what I do is take the first arg, then " ".join() the rest.
Alternative would be to force the user to quote device names which is more typing and a pain in the neck on phones.

File: main.py (hunk 1 above)
Comment (vdbg) [reply main.py:236]: ok

File: main.py (hunk 2 above)
Comment (vdbg) [reply main.py:346]: nice one! thx

**Pull request merged**